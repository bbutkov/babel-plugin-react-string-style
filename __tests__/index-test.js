const babel = require('@babel/core');
const plugin = require('../');

const plugins = ['@babel/plugin-syntax-jsx', plugin];

it('single node', () => {
    const example = `<div style="width:12px"/>`;
    const {code} = babel.transform(example, { plugins });
    expect(code).toMatchSnapshot();
});

it('nested node', () => {
    const example = `<div style="width:12px"><span style="height: 20px"></span></div>`;
    const {code} = babel.transform(example, { plugins });
    expect(code).toMatchSnapshot();
});
