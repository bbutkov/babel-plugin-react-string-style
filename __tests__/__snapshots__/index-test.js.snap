// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nested node 1`] = `
"<div style={
  width: \\"12px\\"
}><span style={
    height: \\"20px\\"
  }></span></div>;"
`;

exports[`single node 1`] = `
"<div style={
  width: \\"12px\\"
} />;"
`;
